packages/@lwc/template-compiler/src/__tests__/fixtures/directive-render-mode/error-dom-manual/metadata.json

```diff
@@ -2,7 +2,7 @@
     "warnings": [
         {
             "code": 1135,
-            "message": "LWC1135: Invalid directive 'lwc:dom' on element. 'lwc:dom' is not supported in Light DOM components.",
+            "message": "LWC1135: Invalid directive 'lwc:dom' on element <p>. 'lwc:dom' is not necessary in Light DOM components.",
```

Comment:
I would have added the component name here, but in the Rollup plugin at least, it looks like it already outputs the filename when logging the error (in `lwc-barebone` anyway).